src/modules/person/api/person.controller.ts

```diff
@@ -101,6 +103,7 @@ export class PersonController {
     }
 
     @Post()
+    @UseGuards(StepUpGuard)
     @HttpCode(HttpStatus.CREATED)
```

Comment:
General comment for all `UseGuards(StepUpGuard)`:
You can decorate the entire controller-class, to add the guard to all routes (only if all of the routes in that controller need it, of course)